docs/userguides/development.md

```diff
@@ -101,6 +139,34 @@ If you configure your application to use a signer, and that signer signs anythin
 Always test your applications throughly before deploying.
 ```
 
+### Distributed Execution
+
+Using only the `silverback run ...` command in a defualt configuration executes everything in one process and the job queue is completely in-memory with a shared state.  In some high volume environments, you may want to deploy your Silverback application in a distributed configuration.
+
+The primary components are the client and workers.  The client handles Silverback events (blocks and contract event logs) and creates jobs for the workers to process in an asynchronous manner.
+
+For this to work, you must configure a [TaskIQ broker](https://taskiq-python.github.io/guide/architecture-overview.html#broker) capable of distributed processing.  For instance, with [`taskiq_redis`](https://github.com/taskiq-python/taskiq-redis) you could do something like this for the client:
```

Comment:
the redis broker has a fundamental issue due to using a pubsub architecture, I think the rabbitmq broker would be better to point out for people to try first

Reply:
I used redis because it was dirt simple to setup locally (and works fine?).  Might take me a bit to get a rabbitmq one together.

Reply:
https://github.com/taskiq-python/taskiq-redis#pubsubbroker-and-listqueuebroker-configuration

Reply:
I'm using `ListQueueBroker` in the examples

Reply:
Okay, I guess this is actually fine. The nice thing is that the redis plugin can be used as both a broker and a result backend